src/components/ChatApp/MessageListItem/generateMessageBubbleUtil.js

```diff
@@ -92,7 +92,12 @@ const generateGifBubble = (
     <MessageContainer key={action + index}>
       <MessageBubble author={message.authorName}>
         <div>
-          <iframe src={gifSource} frameBorder="0" allowFullScreen />
+          <iframe
+            src={gifSource}
+            title="SUSI features GIF"
```

Comment:
SUSI GIF

Reply:
This title was suggested by Akshat Garg. And similarly others.